pkg/controller/add_kubedirectorconfigmap.go

```diff
@@ -0,0 +1,11 @@
+package controller
+
+import (
+	"github.com/bluek8s/kubedirector/pkg/controller/kubedirectorconfigmap"
```

Comment:
That new controller subdirectory (and the name of this file and var names etc.) should just use "configmap", not "kubedirectorconfigmap" ... i.e. the resource we're controlling is just a "configmap" resource. Need to a search-and-replace for kubedirectorconfigmap and KubeDirectorConfigMap to fix that up.

Reply:
Yup, will do